src/technology-consulting/5-communication.md

```diff
@@ -82,6 +71,17 @@ Often, a project or discussion will end without clear roles, responsibilities, o
 
 As dumb as it sounds, I’ve found that one of the key differences between great consultants and average ones is caring - really caring about making the project successful through any means necessary. Most people just go through the motions. If you care a lot, and you’re not ashamed to show it, people will notice. Other people who also care will be attracted to you and respect you, and doors will tend to open as a result.
 
+## Communication Tools
+
+The following the communication tools that should be present on every project. With so much communication technology it important to pick the correct technology for the type of interaction from the receiver you expect. Immediacy, as in how immediate do you need a response, is the governing attribute for each tool. Below are a list of common tools used in today's workplace in order from least immediate to most immediate.
+
+- Wiki
+- Email
+- Chat
+- Meetings
+
+When deciding what tool to use start at the top as it is the most respectful of your recipient's time. Most communications are informational in nature and should documented (like in a wiki) or emailed. Chat replies can be delayed but most modern tools like Slack give the sender the ability to "demand" a recipients attention with notifications. Meetings should only be scheduled when decisions need to be made at that very moment.
```

Comment:
Note in here somewhere (or link to the writing section below) that people will assume email is informational; if you want a response, clearly indicate so.